screens/Home/MyVcs/GetVcModal.tsx

```diff
@@ -13,7 +13,9 @@ export const GetVcModal: React.FC<GetVcModalProps> = (props) => {
     <React.Fragment>
       <GetIdInputModal
         service={props.service}
-        isVisible={true}
+        isVisible={
+          !controller.isAcceptingOtpInput && !controller.isRequestingCredential
```

Comment:
Can we look if this extraction of the logic is possible `stateInAcceptingApplicationID() -> bool`? It looks like the logic is being formulated based on deny list of states, rather it is better to show the dialog based on allowed state.

Reply:
we have used the "isAcceptingUinInput" state to load the modal, tested it and working as expected.